alpine-php-fpm-drupal7/3.4/Dockerfile

```diff
@@ -25,6 +25,7 @@ RUN php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');" &&
 # Install drush and symlink it somewhere useful.
 RUN COMPOSER_HOME=/usr/local/drush \
     composer global require drush/drush 6.7
+RUN drush @none dl registry_rebuild
```

Comment:
Ey joh die hoort daar helemaal niet tussen!

Can you move this line to _under_ the one where we symlink the just-installed drush please?